src/components/Integrations/Shopify/Shopify.js

```diff
@@ -191,14 +194,26 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
 
               <footer className="dp-integration__actions">
                 {backButton}
-                <a
-                  href={_('shopify.connect_url')}
-                  className="dp-button button-medium primary-green"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {_('common.connect')}
-                </a>
+                {!shopifyState.isConnecting ? (
+                  <a
+                    href={_('shopify.connect_url')}
+                    className="dp-button button-medium primary-green"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    onClick={() => {
+                      setShopifyState({ isConnecting: true });
+                    }}
+                  >
+                    {_('common.connect')}
+                  </a>
+                ) : (
+                  <button
+                    disabled
+                    className="dp-button button-medium primary-green button--loading"
```

Comment:
`primary-green` is a generic class? or is only for buttons? because the name means it's generic dont?

Reply:
why `button-medium` have only one `-` and `button--loading` have 2? both are modifiers maybe

Reply:
I got this classes from ui library. I will check again to see if they are correct.

Reply:
They are as in ui library, please refer your concern there. http://cdn.fromdoppler.com/doppler-ui-library/latest/index.html

Reply:
I think that we agreed that modifiers will be added into classes using -- so yes it should be `button--medium` and --`button--primary` instead of `primary--green` right? @GusBaamonde @fcoronelmakingsense thoughts?

Reply:
i think you are right, but if change need to check in all places that use this classes

Reply:
I agree, but I think that in a more detailed refactor for another pull request

Reply:
Si por eso decia, tal vez cambiemos el nombre de la clase a uno copado, manteniendo la clase vieja, hasta que cambiemos todo lo de webapp y ahi deprecamos la clase vieja de la libreria de ui.